src/firebolt/model/V2/instance_type.py

```diff
@@ -1,8 +1,14 @@
 from enum import Enum
+from typing import Any
 
 
 class InstanceType(Enum):
     S = "S"
     M = "M"
     L = "L"
     XL = "XL"
+    UNKNOWN = "UNKNOWN"  # instance type could not be determined
```

Comment:
These are not the only supported Types, we have others in certain cases, SDKs should not break due to this. Please treat them as a string.

Reply:
This is exactly what this PR addresses - not breaking due to unexpected values.